src/framework/grid/footer/_gridFooterCell.scss

```diff
@@ -1,21 +1,19 @@
+
 .grid__footer__cell {
+  padding-left: 8px;
+  padding-right: 8px;
```

Comment:
```
padding-left: 8px;
padding-right: 8px;
```
is being used in `_gridHeader`, `_gridBodyCell`, `_gridFooterCell` and `_stickyGrid`. Could be pulled out.